docs/running.rst

```diff
@@ -126,19 +128,25 @@ Alternatively, to run the simulation in MPI mode:
 
 .. code-block:: none
 
-   $ yank script --yaml=yank.yaml
+   $ mpirun -n # yank script --yaml=yank.yaml
```

Comment:
That was my thought as well. but based on @davidlmobley 's comments regarding his SLRUM server it did not appear to be working. So I added this in response to those comments, but it may be a mater of more debugging first.